pkg/server/health_check.go

```diff
@@ -3,19 +3,33 @@ package server
 import (
 	"go.uber.org/zap"
 
-	"github.com/cmsgov/easi-app/pkg/cedar"
+	"github.com/cmsgov/easi-app/pkg/cedar/cedareasi"
+	"github.com/cmsgov/easi-app/pkg/cedar/cedarldap"
 	"github.com/cmsgov/easi-app/pkg/email"
 )
 
-// CheckCEDARClientConnection makes a call to CEDAR to test it is configured properly
+// CheckCEDAREasiClientConnection makes a call to CEDAR Easi to test it is configured properly
 // this method will panic on failures
-func (s Server) CheckCEDARClientConnection(client cedar.TranslatedClient) {
-	s.logger.Info("Testing CEDAR Connection")
+func (s Server) CheckCEDAREasiClientConnection(client cedareasi.TranslatedClient) {
+	s.logger.Info("Testing CEDAR EASi Connection")
 	// FetchSystems is agnostic to user, doesn't modify state,
 	// and tests that we're authorized to retrieve information
 	_, err := client.FetchSystems(s.logger)
 	if err != nil {
-		s.logger.Fatal("Failed to connect to CEDAR on startup", zap.Error(err))
+		s.logger.Fatal("Failed to connect to CEDAR EASi on startup", zap.Error(err))
+	}
+}
+
+// CheckCEDARLdapClientConnection makes a call to CEDAR LDAP to test it is configured properly
+// this method will panic on failures
+func (s Server) CheckCEDARLdapClientConnection(client cedarldap.TranslatedClient) {
+	s.logger.Info("Testing CEDAR LDAP Connection")
+	// Authenticate is agnostic to user, doesn't modify state,
+	// and tests that we're authorized to retrieve information
+	//TODO standardize this in some way that doesn't use a specific EUAID
+	_, err := client.FetchUserEmailAddress(s.logger, "GZVA")
```

Comment:
we shouldn't use your EUA id here. Uncertain what we should do though.

Reply:
Yeah...

Reply:
I asked Don if there's some sample or test user we could use, we'll see

Reply:
I thought we agreed earlier today to comment out this test (and use "ABCD")